nextflow_schema.json

```diff
@@ -1025,19 +1027,19 @@
                 },
                 "logo_file": {
                     "type": "string",
-                    "default": "docs/images/nf-core-differentialabundance_logo_light.png",
+                    "default": "${projectDir}/docs/images/nf-core-differentialabundance_logo_light.png",
```

Comment:
Looks fine in principle, but these have been working fine for ages- why the sudden need for `projectDir`?

Reply:
I have no idea either, but for some reason now, having the path without `projectDir` causes this error: https://github.com/nf-core/differentialabundance/actions/runs/8416574741/job/23043674976?pr=256